app/main.css

```diff
@@ -33,7 +33,7 @@ progress::-webkit-progress-value {
 
 .main {
   display: flex;
-  max-width: 1024px;
+  max-width: 64rem;
```

Comment:
I'm trying to use rem as much as possible, but if px is important feel free to change it back.